packages/studiocms_core/src/db/tables.ts

```diff
@@ -141,5 +141,7 @@ export const StudioCMSSiteConfig = defineTable({
 		siteIcon: column.text({ optional: true }),
 		loginPageBackground: column.text({ default: 'studiocms-curves' }),
 		loginPageCustomImage: column.text({ optional: true }),
+		enableDiffs: column.boolean({ default: false }),
+		diffPerPage: column.number({ default: 10 }),
```

Comment:
This should probably be `diffsPerPage`?

```suggestion
		diffsPerPage: column.number({ default: 10 }),
```

Reply:
Valid point.  Only downside... its a bit hard to rename a table column after you've already pushed the schema to the database... :sweat_smile: and we would probably need the first time setup stuff ready first... so i can purge the db and reset while updating the schema